boards/heltec_vision_master_e213.json

```diff
@@ -0,0 +1,39 @@
+{
+  "build": {
+    "arduino": {
+      "ldscript": "esp32s3_out.ld",
+      "partitions": "default_8MB.csv"
+    },
+    "core": "esp32",
+    "extra_flags": [
+      "-DBOARD_HAS_PSRAM",
+      "-DHELTEC_VISION_MASTER_E213",
+      "-DARDUINO_USB_CDC_ON_BOOT=1",
+      "-DARDUINO_USB_MODE=0",
+      "-DARDUINO_RUNNING_CORE=1",
+      "-DARDUINO_EVENT_RUNNING_CORE=1"
+    ],
+    "f_cpu": "240000000L",
+    "f_flash": "80000000L",
+    "flash_mode": "qio",
+    "hwids": [["0x303A", "0x1001"]],
```

Comment:
>When on flash mode (Middle button+ boot button) it has a different serial port number than when powered on on my windows computer. Not sure if there's a fix for that.

I'm experimenting with use the two different hardware ids for this device. If nothing else, it seems to make the PlatformIO serial monitor find the second port correctly.
```
      "hwids": [
        [
          "0x303A",
          "0x1001"
        ],
        [
          "0x303A",
          "0x0002"
        ]
````

I'd like to play with this a little bit more on my end and see if we can get an even better solution.